apps/hash-frontend/src/shared/entity-type-entities-context/use-entity-type-entities-context-value.tsx

```diff
@@ -64,6 +63,8 @@ export const useEntityTypeEntitiesContextValue = (params: {
       constrainsValuesOn: { outgoing: 255 },
       inheritsFrom: { outgoing: 255 },
       isOfType: { outgoing: 1 },
+      hasLeftEntity: { outgoing: 1, incoming: 1 },
+      hasRightEntity: { outgoing: 1, incoming: 1 },
```

Comment:
This is going to significantly slow loading in places where we fetch a lot of entities and don't currently care about linked entities (e.g. entities tab on the Entity Type page, the table on the `/entities` route) – I think we should make this an argument can be passed as needed.